docs/EN_US/ContainerizedHPCC/ContainerizedMods/AzureDeployChapter.xml

```diff
@@ -0,0 +1,645 @@
+eclservices ClusterIP    10.0.44.11    &lt;none&gt;          8010/TCP        11m
+<emphasis role="bold">eclwatch    LoadBalancer 10.0.21.16    12.87.156.228   8010:30190/TCP  11m</emphasis>
+kubernetes  ClusterIP    10.0.0.1      &lt;none&gt;          443/TCP         4h28m
+mydali      ClusterIP    10.0.195.229  &lt;none&gt;          7070/TCP        11m</programlisting>
+
+      <para>Use the EXTERNAL-IP address listed for the ECLWatch service. Open
+      a browser and go to http://&lt;external-ip&gt;:8010/. For example in
+      this case, go to http://12.87.156.228:8010. If everything is working as
+      expected, the ECLWatch landing page will be displayed.</para>
+    </sect2>
+
+    <sect2 id="AZUninstall_YourCluster">
+      <title>Uninstall Your Cluster</title>
+
+      <para>When you have completed using your HPCC Systems instance, unless
+      your project/job/work is intended to persist. It is a good idea to
+      remove all components deployed to the cloud.</para>
+
+      <para>In completely doing so you would remove the resource group you
+      created, which will irreversibly destroy any pods, contents, or work on
+      there, including all customizations. In addition if you have any jobs
+      that are in the ephemeral storage they will also be deleted. So before
+      undertaking this step carefully consider that this is permanent and make
+      certain that you really intend to remove everything. If unsure you might
+      better off to just leave it.</para>
+
+      <sect3>
+        <title>Stopping Your HPCC Systems Cluster</title>
+
+        <para>To shut down your HPCC Systems cluster, you would issue the helm
+        uninstall command.</para>
+
+        <para>Using the Azure cloud shell, enter:</para>
+
+        <programlisting>helm list</programlisting>
+
+        <para>Enter the helm uninstall command using your clusters name as the
+        argument, for example:</para>
+
+        <programlisting>helm uninstall myhpcccluster</programlisting>
+
+        <para>This will remove the HPCC Systems cluster named
+        &lt;myhpcccluster&gt; you had previously deployed. Which if you had
+        configured persistent storage, your data could still exist. To
+        complete the removal of all your components, and ensure you are not
+        continuing to incur any costs, remove your resource group.</para>
+      </sect3>
+
+      <sect3>
+        <title>Removing the Resource Group</title>
+
+        <para>As stated earlier removing the resource group will irreversibly
+        destroy any pods, contents, or any other work stored on there. Once
+        again, please carefully consider these actions, before removing the
+        resource group. Once removed it can not be undone. </para>
+
+        <para>To remove the entire resource group <emphasis>rg-hpcc</emphasis>
+        which we created earlier, using the Azure cloud shell issue the
```

Comment:
I wouldn't mention cloud shell on this documentation. Any bash shell can do the same thing. It's ok to mention that you are using it in the beginning. But after that, it doesn't have any merit.